### PR TITLE
Depth graph

### PR DESCRIPTION
**Pull request by paronicholas**
- [x] Top-level README “Table of Contents” is updated
- [x] Feature tasks for this challenge are completed
- [x] Unit tests written and passing
    - [x] “Happy Path” - Expected outcome
    - [x] Expected failure
    - [x] Edge Case (if applicable/obvious)
- [x] README for this challenge is complete
    - [x] Summary, Description, Approach & Efficiency, Solution, Resources
    - [x] Link to code
    - [x] Link to tests

### CODE REVIEW

File: challenge-readme/dfg.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Create a static method which returns a Set of the Graph vertices in a depth first orientation based off of any root vertex added.
+
+### Links
+* src/main/graph
+  * [Graph.java](../code401challenges/src/main/java/graph/Graph.java)
+* src/test/graph
+  * [GraphTest.java](../code401challenges/src/test/java/graph/GraphTest.java)
+
+### Challenge
+Create a  method called `depthFirst(Graph<F> graph, Vertex<F> root)` which changes returns a set of the nodes in breadth order (layer-by-layer) from the specified root.
```
Comment (evanslaton) [inline challenge-readme/dfg.md:11]: This is describing the breadth first traversal
